hack/dockerized

```diff
@@ -148,8 +147,8 @@ fi
 
 # Run the command
 test -t 1 && USE_TTY="-it"
-EXEC_ENV_VARS="--env KUBEVIRT_CRI=${KUBEVIRT_CRI}"
-if ! $KUBEVIRT_CRI exec $EXEC_ENV_VARS ${USE_TTY} ${BUILDER}-bazel-server /entrypoint.sh "$@"; then
+EXEC_ENV_VARS=("--env" "KUBEVIRT_CRI=${KUBEVIRT_CRI}")
```

Comment:
Why is this change needed?

Reply:
It is needed because of spaces introduced by using `podman` with the `--remote` and `--url` arguments here https://github.com/kubevirt/kubevirt/blob/9c31510e3f22991c7cfcc232ef67a4b6d16134b1/hack/common.sh#L10

Although I am sure why the `$KUBEVIRT_CRI` as env variable is needed here in the first place. Can you explain? @iholder-redhat

Reply:
I think the purpose of `KUBEVIRT_CRI` is to explicitly specify a CRI.
IOW, if you have both Podman and Docker installed and you want to explicitly state that you want to use one of them.

Reply:
Yes, but why does the `entrypoint.sh` of bazel-server need `KUBEVIRT_CRI`?

Reply:
@0xFelix Hi. I think its needed for the other bash script file provided from the outer context and that is invoked by `"$@"` after `entrypoint.sh` exeution.

Reply:
Hi @enp0s3, sorry, I see! `entrypoint.sh` runs a build script. Are podman and docker included in the builder image? Sorry, I am confused.

Reply:
@0xFelix Lets remove it, but please run `make` and `make test` to make sure we don't break anything. Thanks!

Reply:
@enp0s3 `make` and `make test` run sucessfully after removing `EXEC_ENV_VARS`. I updated the PR.